tslint.json

```diff
@@ -166,6 +166,78 @@
 					"@angular/*",
 					"rxjs/*"
 				]
+			},
+			{
+				"target": "**/vs/platform/*/common/**",
+				"restrictions": [
+					"vs/nls",
+					"**/{vs,sql}/base/common/**",
+					"**/{vs,sql}/base/parts/*/common/**",
+					"**/{vs,sql}/platform/*/common/**"
+				]
+			},
+			{
+				"target": "**/vs/platform/*/test/common/**",
+				"restrictions": [
+					"assert",
+					"sinon",
+					"vs/nls",
+					"**/{vs,sql}/base/common/**",
+					"**/{vs,sql}/platform/*/common/**",
+					"**/{vs,sql}/platform/*/test/common/**"
+				]
+			},
+			{
+				"target": "**/vs/platform/*/browser/**",
+				"restrictions": [
+					"vs/nls",
+					"vs/css!./**/*",
+					"**/{vs,sql}/base/{common,browser}/**",
+					"**/{vs,sql}/base/parts/*/{common,browser}/**",
+					"**/{vs,sql}/platform/*/{common,browser}/**"
+				]
+			},
+			{
+				"target": "**/vs/platform/*/node/**",
+				"restrictions": [
+					"vs/nls",
+					"**/{vs,sql}/base/{common,browser,node}/**",
+					"**/{vs,sql}/base/parts/*/{common,browser,node}/**",
+					"**/{vs,sql}/platform/*/{common,browser,node}/**",
+					"!path" // node modules (except path where we have our own impl)
```

Comment:
i've seen comments causing errors in .json files

Reply:
it depends on what the json file is used by, the tslint file is pulled in by the tslinting library which supports the jsonc format for json files, so its not an issue, which allows comments.